network/load-balancer/reference-content/acls.mdx

```diff
@@ -0,0 +1,116 @@
+
+## Using ACLs to filter traffic's access to the backend
+
+Below, we show how to achieve a number of different filtering objectives with ACLs
+
+### Only allow traffic from certain IPs
+
+In this case, we only want traffic from source IPs `51.158.99.227` and `151.115.78.39` to be allowed to pass to the backend.
+
+#### Option 1
+Construct two ACLs:
+- **ACL Priority 1**: `Allow` - `Enable condition` - `IF MATCH` - `IP filter`: `51.158.99.227`, `151.115.78.39`. This ACL will allow traffic from the two specified IPs to pass directly to the backend. All traffic not from these IPs will pass to the next ACL.
+- **ACL Priority 2**: `Deny` - `Disable condition`. This ACL will deny access to all remaining traffic.
+
+#### Option 2
+Construct one ACL:
+- **ACL Priority 1**: `Deny` - `Enable condition` - `IF NOT MATCH` - `IP filter`: `51.158.99.227`, `151.115.78.39`. This ACL will reject any traffic **not** from the two specified IPs. Traffic from the specified IPs will, by default, pass to the backend since there are no further ACLs in the list.
+
+### Reject traffic from certain IPs, and with certain HTTP header values.
+
+In this case, we want to reject traffic from source IPs `51.158.99.227` and `151.115.78.39`, and also reject traffic with either of these IP addresses in the `X-Forwarded-For` HTTP header. All other traffic should be allowed access.
+
+Construct two ACLs:
+- **ACL Priority 1**: `Deny` - `Enable condition` - `IF MATCH` - `IP filter`: `51.158.99.227`, `151.115.78.39`. This ACL will reject traffic from the two specified IPs. All traffic not from these IPs will pass to the next ACL.
+- **ACL Priority 2**: `Deny` - `Enable condition` - `IF MATCH` - `HTTP filter - Header` - `Header name: X-Forwarded-For` `Filters`: `51.158.99.227`, `151.115.78.39`. This ACL will reject traffic that has the specified IP values in the specified header. All traffic without these values in that header, which hasn't already been rejected by the previous ACL, will by default pass to the backend since there are no further ACLs in the list.
+
+<Message type="note">
+
+  In this example, you could switch the priority order of the ACLs without any effect.
+
+</Message>
+
+### Reject traffic from certain IPs, and with HTTP header values, unless it has a specific HTTP request path
+
+In this case, we want to achieve the same as with our previous example, **however**, if that same traffic is requesting an HTTP request path that begins with `/catalog` we want to allow access nonetheless.
+
+Construct three ACLs:
+- **ACL Priority 1**: `Allow` - `Enable condition` - `IF MATCH` - `HTTP filter` `Path begins with`: `/catalog`. This ACL will allow any traffic with this at the beginning of its HTTP header to pass directly to the backend. All traffic without this header path will pass to the next ACL.
```

Comment:
> This ACL will allow any traffic with this at the beginning of its HTTP header to pass directly to the backend. All traffic without this header path will pass to the next ACL.

This ACL will allow any HTTP request with `/catalog` at the beginning of its URI path to pass directly to the backend. All HTTP requests without `/catalog` at the beginning of its URI path will pass to the next ACL.